src/java/org/apache/cassandra/repair/autorepair/AutoRepair.java

```diff
@@ -167,6 +168,11 @@ public void repair(AutoRepairConfig.RepairType repairType)
             logger.debug("Auto-repair is disabled for repair type {}", repairType);
             return;
         }
+        if (!Guardrails.mixedRepairsEnabled.isEnabled(null))
+        {
+            logger.debug("Cannot run repair when nodes in the cluster have different versions.");
```

Comment:
Could we use a NoSpamLogger to log it as _Info_ every 5 minutes? The reason is without any logging or metrics, it would become difficult for admins to diagnose why the repair has not been running. A new metric here would also be fine.